_gsocproposals/proposal_ATLASEigen.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Hi Stewart,
We made some changes to the internal structure of the repository. Could you rebase the PR? For instance, your PR should go in another directory  (`_gsocproposals/2018`). Thank you!